ViewModel/EnvironmentAndExtendStoreUuid.php

```diff
@@ -138,7 +138,7 @@ private function getScopedConfigValue(string $configPath): string
     {
         $scopeCode = $this->storeManager->getStore()->getCode();
         $scopeType = ScopeInterface::SCOPE_STORES;
-        return $this->scopeConfig->getValue($configPath, $scopeType, $scopeCode);
+        return $this->scopeConfig->getValue($configPath, $scopeType, $scopeCode) ?: '';
```

Comment:
Doesn't getValue return null anyways, if there is no value or the path isn't valid?

Reply:
it was returning null and causing an error